File: java_backup/my java/tapabrata/sum_of_digits.java
class sum_of_digits
{int c,c1,n;
void main()
{c=0;
c1=sum(n);
System.out.println(c1);
}
int sum(int n1)
{if(n1==0)
return c;
else
{
c=c+n1%10;
return sum(n1/10);
}
}
}

File: 6th_sem/MATLAB/AN SIr till 27-Mar-2018/sum_of_digits.m
%function s= sum_of_digits(n) : Function to calculate sum of digits in any
%integer n
 function s=sum_of_digits(n)
   s=0;
    while n ~=0
    d=mod(n,10);
    s=s+d;
    n=fix(n/10);
    end
   end
